config.go

```diff
@@ -21,6 +21,6 @@ type Config struct {
 	OAuthAuthUrl     string `envconfig:"OAUTH_AUTH_URL" default:"https://getalby.com/oauth"`
 	OAuthTokenUrl    string `envconfig:"OAUTH_TOKEN_URL" default:"https://api.getalby.com/oauth/token"`
 	OAuthServerPort  string `envconfig:"OAUTH_SERVER_PORT" default:"8080"`
-	DatabaseUri      string `envconfig:"DATABASE_URI"`
+	DatabaseUri      string `envconfig:"DATABASE_URI" default:"wb.db"`
```

Comment:
What does `wb` stand for? I would rather call it `nostr-wallet-connect.db` to be more verbose in what it contains.